examples/android-app/cubicsdk/src/main/proto/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
I agree

Reply:
1. Now I have made it so that the connection dialog does not close. I have thoughts on how to connect the connection indication with the talk button, I will try to implement it.

2. The fall of the application corrected

3. Did you mean the settings for the recording model? Yes, I think to display the selected model as a settings icon.

Reply:
1. ok.
2. great!
3. see my other response.

Reply:
Added a script in gradle for loading proto files from the repository